demos/src/css/chartAuthoring-demo.css

```diff
@@ -27,7 +31,3 @@
     clear: both;
     width:50em;
 }
-
-.floec-dataEntryPanel li {
```

Comment:
Is there a reason to remove this css? With this change, the data entry form now has the listing dots in front of each row:
![screen shot 2015-10-30 at 10 38 22 am](https://cloud.githubusercontent.com/assets/572877/10848593/15238a66-7ef3-11e5-9fac-c754b5aa4aa9.jpeg)